src/procedures/ModifyTieredStoData.ts

```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
   return undefined;
 };
 
+/**
+ * Procedure to modify data associated to a Tiered STO.
+ * Transactions include times, funding type, oracles, tiers, limits, and addresses
+ */
 export class ModifyTieredStoData extends Procedure<ModifyTieredStoDataProcedureArgs> {
   public type = ProcedureType.ModifyTieredStoData;
 
+  /**
```

Comment:
Delete the empty lines between the items

Reply:
Also, after the items and before the fail notes, add 

`Only transactions that will effectively present changes will be submitted`